docs/en/development/contributing-doc.md

```diff
@@ -0,0 +1,75 @@
+# Document Contribution Guide
+
+Thank you for your interest in Layotto!
+
+This document describes how to modify/add documents. Documentation for this repository is written in Markdown.
+
+## Document Path description
+
+Documents are stored in the docs/ directory, where docs/en stores English documents and docs/zh stores Chinese documents.
```

Comment:
It looks better to add single quotes  to path description?  `docs/` `docs/en` `docs/zh`